staging/kos/cmd/attachment-tput-driver/main.go

```diff
@@ -1297,3 +1309,33 @@ func withRetries(thunk func() bool, minWait, maxWait time.Duration, stopCh <-cha
 	}
 	return true
 }
+
+func printCSVOpsTimes(opsTimes map[uint64]threadOpsSchedule, rate float64, distribution, outputDir string) (err error) {
+	opsTimesFilename := filepath.Join(outputDir, "attachments-ops-times.csv")
+	var opsTimesCSVFile *os.File
+	opsTimesCSVFile, err = os.Create(opsTimesFilename)
+	if err != nil {
+		return
+	}
+	defer func() {
+		cerr := opsTimesCSVFile.Close()
+		if err == nil {
+			err = cerr
+		}
+	}()
+	header := fmt.Sprintf("\"distribution: %s, rate: %g\"\n\"cell(i, j > 0) = dt in seconds at which ops j for thread i must take place wrt start of operations\"\n", distribution, rate)
+	if _, err = opsTimesCSVFile.Write([]byte(header)); err != nil {
+		return
+	}
+	for thd := 0; thd < len(opsTimes); thd++ {
+		thdTimes := opsTimes[uint64(thd)]
+		csvRow := fmt.Sprintf("thd %d", thd+1)
+		for _, dtNanos := range thdTimes {
```

Comment:
"nanos" is a confusing thing to put in the variable name here.  The fact that they are nanoseconds is not used by the code here, and generally is almost never needed by code that manipulates `time.Duration` values (which is what these are).  How about "thdTime" or "dt" or "duration" or "durationTillStart" instead of "dtNanos"?

Reply:
Switched to `dt` in in49677ba .